<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPV Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial connection</string>
    </property>
    <widget class="QPushButton" name="buttonUartRefresh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxUart">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonUartConnect">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="UartStatusIndicator">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnected</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonTest">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGetStatus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRequestChannelFill">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Request Channel Fill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStartPlaying">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Playing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStopPlaying">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Playing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonInitSPV">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Init Channels</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRewind">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rewind</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClearChannels">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Channels</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open m-code</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
